_data/retail.yml

```diff
@@ -72,6 +72,12 @@ websites:
       img: groupon.png
       tfa: No
 
+    - name: Home Depot
+      url: http://www.homedepot.com
```

Comment:
When using HTTPS I receive a certificate error saying: 

> There are issues with the site's certificate chain (net::ERR_CERT_COMMON_NAME_INVALID).

Is anyone else experiencing this?